packages/useraccounts-core/lib/methods.js

```diff
@@ -89,8 +88,14 @@ if (Meteor.isServer) {
                 }).data;
 
                 if (!apiResponse.success) {
-                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification,
-                      apiResponse['error-codes'] ? apiResponse['error-codes'].join(", ") : "Unknown Error.");
+                    try {
+                        var msg = apiResponse['error-codes'].join(", ");
+                    } catch(e) {
+                        console.error('Exception Caught: ',e);
+                        var msg = "Unknown Error.";
+                    }
+
+                    throw new Meteor.Error(403, AccountsTemplates.texts.errors.captchaVerification, msg);
```

Comment:
This is very strange - it shouldn't even be hitting this part of the code since we're not using recaptcha. Could we try putting in some logging to try to figure out what's going on?